h2o-r/h2o-package/R/frame.R

```diff
@@ -1926,10 +1926,15 @@ h2o.setLevels <- function(x, levels, in.place = TRUE) .newExpr("setDomain", chk.
 #' \dontrun{
 #' library(h2o)
 #' h2o.init(ip <- "localhost", port = 54321, startH2O = TRUE)
-#' australia_path <- system.file("extdata", "australia.csv", package = "h2o")
```

Comment:
No need to change the import commands. This method works in R and points to a csv that is packaged with the R client.